ui-components/src/locales/general.json

```diff
@@ -1,8 +1,8 @@
 {
   "about": {
     "body1": "We know that finding housing that meets your needs can be difficult and frustrating. Detroit Home Connect is your helping hand to find a new place to call home.",
-    "body2": "Detroit Home Connect is a new City of Detroit service that provides you a central first step in finding housing in Detroit that meets your affordability and household needs. You can understand your eligibility for rental units by exploring options based on your family size, age, and income. The City of Detroit’s Housing and Revitalization Department has collaborated with Google.org to create Detroit Home Connect. The design and features of the website is based on feedback and insight from area residents and community-based organizations.", 
-    "moreInfoContact": "For more information, please contact Safiya Merchant at safiya.merchant@detroitmi.gov.",
+    "body2": "Detroit Home Connect is a new City of Detroit service that provides you a central first step in finding housing in Detroit that meets your affordability and household needs. You can understand your eligibility for rental units by exploring options based on your family size, age, and income. The City of Detroit’s Housing and Revitalization Department has collaborated with Google.org and Exygy to create Detroit Home Connect. The design and features of the website is based on feedback and insight from area residents, community-based organizations, property managers, and property owners.", 
```

Comment:
Since the translations for these strings already exist in the other languages, if we just change it in the English file, the other translations will be out of date but we won't know. We should remove these keys in the foreign key files so we know that they are missing.